.github/workflows/label-analysis.yml

```diff
@@ -13,39 +13,45 @@ jobs:
     if: github.triggering_actor != 'dd-devflow[bot]'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: "3.11"
-          cache: "pip"
-      - name: Install dependencies
+      - name: Check team assignment
         run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements.txt
-          pip install -r tasks/libs/requirements-github.txt
-      - name: Run PR check for team assignment label
-        env:
-          BRANCH_NAME: ${{ github.ref_name }}
-          PR_ID: ${{ github.event.pull_request.number }}
-        run: |
-          inv -e lint-teamassignment
+          labels="${{join(github.event.pull_request.labels.*.name, ' ')}}"
+          for label in $(echo "$labels" | tr ' ' '\n'); do
```

Comment:
nit: I don't think you need th ` | tr` part
`for label in $labels` should work out of the box

Reply:
Yes, you are right, changing, thanks